<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueueEnter</class>
 <widget class="QMainWindow" name="QueueEnter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>701</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="title">
     <string>功能区</string>
    </property>
    <widget class="QPushButton" name="enterButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>入队</string>
     </property>
    </widget>
    <widget class="QLabel" name="EnterLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>入队元素：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="enterLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="OutLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>出队元素：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="outLineEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="outButton">
     <property name="geometry">
      <rect>
       <x>253</x>
       <y>80</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>出队</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>17</y>
       <width>20</width>
       <height>111</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>17</y>
       <width>20</width>
       <height>111</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="explainButton">
     <property name="geometry">
      <rect>
       <x>462</x>
       <y>57</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>说明</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>57</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>701</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>演示区</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>308</x>
       <y>83</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>291</x>
       <y>122</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>55</y>
       <width>23</width>
       <height>22</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_1">
     <property name="geometry">
      <rect>
       <x>395</x>
       <y>54</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>435</x>
       <y>81</y>
       <width>22</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>456</x>
       <y>123</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>291</x>
       <y>156</y>
       <width>22</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="geometry">
      <rect>
       <x>311</x>
       <y>196</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>352</x>
       <y>222</y>
       <width>23</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>455</x>
       <y>156</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>438</x>
       <y>196</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>398</x>
       <y>220</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>260</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>↑</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>220</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>↗</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>160</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>→</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>125</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>→</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>260</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>↑</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>↘</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>↓</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>↓</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>↙</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>120</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>←</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>160</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>←</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>220</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>↖</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>→ ：队尾位置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;宋体&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;→&lt;/span&gt; ：队首位置&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>341</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>入队算法区</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>25</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>45</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>65</y>
       <width>411</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>85</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>105</y>
       <width>381</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>125</y>
       <width>371</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>145</y>
       <width>371</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>470</y>
      <width>351</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>出队算法区</string>
    </property>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>66</y>
       <width>411</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>46</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>126</y>
       <width>371</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>86</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>26</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>146</y>
       <width>371</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>106</y>
       <width>381</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>735</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
